cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -137,9 +137,9 @@ def test_get_query_count(self, branching_factor, chapter_queries, section_querie
             self.client.get(reverse_usage_url('xblock_handler', self.populated_usage_keys['problem'][-1]))
 
     @ddt.data(
-        (1, 26),
-        (2, 28),
-        (3, 30),
+        (1, 30),
+        (2, 32),
+        (3, 34),
```

Comment:
I think this is modulestore lazy loading again: we're accessing `user_partitions` on the course descriptor and `group_access` on the XBlock, which we weren't doing before.